ee/vellum_cli/tests/test_pull.py

```diff
@@ -94,7 +127,7 @@ def test_pull__sandbox_id_with_no_config(vellum_client):
 
     # WHEN the user runs the pull command with the workflow sandbox id and no module
     runner = CliRunner()
-    result = runner.invoke(cli_main, ["pull", "--workflow-sandbox-id", workflow_sandbox_id])
+    result = runner.invoke(cli_main, ["pull", "workflows", "--workflow-sandbox-id", workflow_sandbox_id])
```

Comment:
Main goal of the PR. Note that removing it from `vellum pull` is a breaking change and we'll need to update our UI